code/parachain/frame/pallet-multihop-xcm-ibc/src/lib.rs

```diff
@@ -106,12 +120,12 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 
 	#[pallet::storage]
-	#[pallet::getter(fn chain_id_to_miltihop_route_path)]
+	#[pallet::getter(fn route_id_to_miltihop_path)]
 	pub type ChainIdToMiltihopRoutePath<T: Config> = StorageMap<
 		_,
 		Blake2_128Concat,
-		u128, //chain id
-		BoundedBTreeSet<(ChainInfo, BoundedVec<u8, T::ChainNameVecLimit>), T::MaxMultihopCount>, /* route to forward */
+		u128, /* chain id */
+		BoundedVec<(ChainInfo, BoundedVec<u8, T::ChainNameVecLimit>), T::MaxMultihopCount>, /* route to forward */
```

Comment:
I'd suggest creating separate types for that, like: 
```
type ChainId = u128;
type Route = BoundedVec<(ChainInfo, BoundedVec<u8, T::ChainNameVecLimit>), T::MaxMultihopCount>;
```

Reply:
type ChainId = u32;

Reply:
Thanks. I will create pr with this changes.